### DISCUSSION TITLE
Should the custom block shape addon change actual block positioning?

### DISCUSSION THREAD

**Discussion by lisa-wolfgang** [ScratchAddons/ScratchAddons#1836] [Discussion]
(Regarding #1773)
The padding size option brings up the issue of the experience of users without the addon enabled. Especially in collaborative contexts, someone with a small padding setting could group the scripts really close together, which would appear as overlapping for people with a larger padding setting.
What would be the best way to remedy this issue? We could:
- just let it be and display a notice in the addon settings
- render the positions (including comments) according to the padding setting but store them with normal padding positioning
- somehow implement it where the vertical distance is always the same
- use a better method not listed here?

**Comment by Endermaster3690** [comment]
i would do
> * just let it be and display a notice in the addon settings

**Comment by Mr-Programmer-777** [comment]
>     * just let it be and display a notice in the addon settings
Yeah.
